.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+
+Therefore: 
+- Do not use internal lingo 
+- Do not assume the reader can access the application you are working on - include screenshots instead.
+- Do not assume the reader can access the repository you are working in - include code snippets instead (if possible). 
+
+<!--- Be specific and unambigious -->
+#### The good issue is specific & unambigious
+<!--- Do point to relevant files if you know them, don't just say "there's some css files". 
+- Facts first opinions second. -->
+
+The wording of an issue should be as specific and unambigious as possible. This will help the reader understand exactly what you mean. This can be done in many ways. Therefore we have compiled some examples to demonstrate:
+
+_Example #1: referring to files_ 
+
+If you are referring to a file in the repository do not refer to it as `_typography.css` use the full path instead: `libs/designsystem/src/lib/scss/base/_typography.scss`.
+While it might be obvious for you and experienced contributors, what and where `_typography.css` is; it is not obvious for newcomers. 
+
+If you already have refered to the full path earlier in the issue, it can be okay to refer to the file as `_typography.css`.
+
+_Example #2: writing reproducible steps for bug reports_
+
+The steps should be unambigously specific. This can be done for example by changing the step: 
+> 2. Type some text 
+
+to: 
+
+> 2. Type: 'text'
+ 
+There could be something with the _exact_ way you are doing it, that triggers the behaviour you are experiecing. 
+
+_Example #3: using code snippets to explain what and how_
+
+When reporting a bug or similar do not write something along the lines of: 
+> Whenever i use a `kirby-button` with attention level 1 on the frontpage, a warning is thrown
+
+Specify exactly where and how the button is inserted with a code snippet, along with exact details of the warning and where it can be found:
+> Inserting a `kirby-button` with attention level 1 as shown in the code-snippet, results in the following warning being logged in the browser console: `Warning goes here`.
+
+The latter is pure maintainer poetry. 
+
+These are but three examples of how issues can be specific and unambigious. But always keep in the back of your mind how you can be more specific and unambigious when writing issues. 
+
+#### The good issue follows the template 
+<!-- - Follow the templates! - they are as is to help you remember which information we're looking for -->
+Do not delete the template when creating an issue. Use it. 
+
+The issue templates exists to ease collaboration by providing structure and reminding you of which information other contributors are looking for. 
+They also provide helpful checklists that helps you follow [the process (INSERT PROPER LINK HERE)](#the-process) and lets others know how far you are. 
+
+If you experience that no template lives up to your specific need; well there is a good opportunity to contribute. Create an issue for a new template to be created! 
```

Comment:
Consider "matches your specific need" instead of "lives up to"

Reply:
Should be solved by 6ede0331559a38a46246aa6f703ae4b697ba1ae9